### DISCUSSION TITLE
Anybody using G.O. with Ardour on Windows?

### DISCUSSION THREAD

**Discussion by EdStauff** [GrandOrgue/grandorgue#1748] [Help]
Is anyone using GrandOrgue with Ardour on Windows?

Ardour is my new DAW since Cakewalk by Bandlab got discontinued, and I need it to play nice with G.O.

They don't seem to want to share the audio hardware.  When Ardour starts its audio engine, G.O. goes silent until I stop Ardour's audio engine and hit G.O.'s panic button.  I tried using Jack, but G.O. didn't like it.

At the very least, I need to play G.O. from Ardour; I can have G.O. record its audio and then import that into Ardour.  The MIDI events are getting sent & received just fine, but Ardour doesn't want to play at all without its audio engine turned on, and then I can't hear G.O.  That's a problem because I need to clean up the MIDI tracks in Ardour, which I can't do without hearing G.O.

Suggestions, anyone?  (That don't involve spending money.)

Thank you,

-- Ed

**Comment by oleg68** [comment]
Go likes Jack. Use Jack Native output for better integration.

**Reply by larspalo** [reply]
Even though I've never used Jack on Windows I'd say the same as Oleg, use Jack with GO for anything more advanced than running it as a stand-alone application, but remember to explicitly set it up in the File->Settings->Audio tab and of course use the same samplerate (and other settings) that Jack uses.

**Comment by midimusic** [comment]
This is an interesting subject. I have just installed Ardour on windows for the first time. I have some audio loaded into it which plays as it should.

Now when I run GO the very first note plays and then there is silence from GO but Ardour continues playing.

On reading some of the documentation Ardour claims to "take over" the audio system. They also state that they discourage the use of jack (which in my experience is not needed by and does not work well on windows so I would not use it).

However the "takeover" is very far from complete because eplayOrgan still plays while Ardour is playing. I am wondering what exactly it takes over and why it needs to take over anything at all.

Comments from other Ardour users would be welcome. 

csw900
